cmd/crictl/main.go

```diff
@@ -220,6 +242,20 @@ func main() {
 			Aliases: []string{"D"},
 			Usage:   "Enable debug mode",
 		},
+		&cli.BoolFlag{
+			Name:  "enable-tracing",
+			Usage: "Enable OpenTelemetry tracing.",
+		},
+		&cli.IntFlag{
+			Name:  "tracing-sampling-rate-per-million",
+			Usage: "Number of samples to collect per million OpenTelemetry spans. Set to 1000000 or -1 to always sample.",
+			Value: -1,
+		},
+		&cli.StringFlag{
+			Name:  "tracing-endpoint",
+			Usage: "Address to which the gRPC tracing collector will send spans to.",
+			Value: "0.0.0.0:4317",
```

Comment:
@saschagrunert, would it make sense to make this use `127.0.0.1` here? Especially since we are trying to fix such things in CRI-O, for example.

Reply:
Fixed :+1: